geospaas/vocabularies/tests.py

```diff
@@ -263,16 +274,18 @@ class CommandsTests(TestCase):
     def test_commands(self, instrument_mock, vdr_mock, tdr_mock, sck_mock,
             project_mock, platform_mock, location_mock, iso_mock, hdr_mock,
             datacenter_mock, parameter_mock):
+        default_options = {'force':False, 'no_color':False, 'pythonpath':None, 'settings':None,
```

Comment:
there are more options here than what is handled in the management command. Can you explain with a comment? Why `no_color`, for example?